src/RulesEngine/Models/ImplicitObject.cs

```diff
@@ -4,16 +4,11 @@
 
 using System.Diagnostics.CodeAnalysis;
 
-using System.Diagnostics.CodeAnalysis;
-
 namespace RulesEngine.Models
 {
     [ExcludeFromCodeCoverage]
     public class ImplicitObject
     {
-
-        #region Implicit Operators
```

Comment:
Do we not want region blocks?

Reply:
the region wraps one block of code.  in my opinion, it isn't necessary